prime-router/settings/staging/0158-al-phl-and-baptist-onboarding.yml

```diff
@@ -0,0 +1,215 @@
+# To submit this to localhost, run:
+# ./prime multiple-settings set --input ./junk/flexion/AL/Settings/al-phl-staging-l.yml
+# ./prime multiple-settings set --env staging --input ./junk/flexion/AL/Settings/al-phl-staging.yml
+#
+# Description:
+#   These following setting are for Louisiana Public Health Lab (al-phl) sender and receivers
```

Comment:
This should say Alabama Public Health Lab

Reply:
Thank you.